### DISCUSSION TITLE
发现B站评论区一个bug

### DISCUSSION THREAD

**Discussion by CKylinMC** [the1812/Bilibili-Evolved#2248] [综合讨论 / General]
视频下评论区加载的最后一条评论会无限复制无限刷出来，但是其实只有那一条...
![image](https://user-images.githubusercontent.com/11648014/131970217-d8d9244e-f389-4c53-b346-f261cfe47730.png)

我一开始想怎么会这么离谱一分钟这么多条，然后我去动态评论区看就一条，回来刷新再看bug还在

和BE脚本无关...仅吐槽

av292846461

**Comment by the1812** [comment]
这个无限滚动评论区, 推出以来质量一直奇差(

**Reply by CKylinMC** [reply]
还不如之前的翻页的...

**Comment by u2shana** [comment]
https://greasyfork.org/zh-CN/scripts/403348
推荐用这个变回旧版
